client_plugin/utils/refcount/refcnt.go

```diff
@@ -447,10 +449,15 @@ func (r *RefCountsMap) syncMountsWithRefCounters(d drivers.VolumeDriver) {
 			}
 		} else {
 			if cnt.count == 0 {
-				// volume unmounted AND refcount 0.  We should NEVER get here
-				// since unmounted and recount==0 volumes should have no record
-				// in the map. Something went seriously wrong in the code.
-				log.WithFields(f).Panic("Internal failure: record should not exist. ")
+				// This volume may be attached to the VM but not mounted as no containers
+				// are using it. A detach is attempted and the entry under the mount root is
+				// removed.
+				err := d.DetachVolume(vol)
+				if err != nil {
+					log.Warning("Failed to detach %s - manual recovery may be needed", vol)
```

Comment:
That could be a release note for the user to check and detach disks  and otherwise ignore the once only warning.